soccer/planning/RRTPlanner.cpp

```diff
@@ -137,15 +139,24 @@ Planning::InterpolatedPath* RRTPlanner::run(
 	{
 		// FIXME: without these two lines, an empty path is returned which causes errors down the line.
 		path->points.push_back(start);
+		path->times.push_back(0);
+		path->vels.push_back(Geometry2d::Point(0,0));
 	}
 	return path;
 }
-
+Planning::InterpolatedPath update(
```

Comment:
Well I wrote this like 2 months ago 8), but I think I thought the planner could use the information of the previous path we were on to select a good point to start planning the new path from.

Or we could have the logic for that done in Robot or something. I guess it just depends on what we want. I think at that point too we didn't have the CompositePath so it was just going to add splice it on using the interpolatedPath